engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/Name.scala

```diff
@@ -799,7 +799,7 @@ object Name {
          |""".toSingleLine
 
     /** @inheritdoc */
-    override def children: List[IR] = List()
+    override def children: List[IR] = Nil
```

Comment:
Isn't `List()` and `Nil` idempotent?

Reply:
It's not. The former is a method call, the latter is an object reference.

Reply:
Method call that returns the same object reference... that's unlikely to have a difference. Such a (small) methods are inlined almost for sure.

Reply:
It doesn't do any harm, does it? Plus then one does not need to think if this has been inlined or not.